src/module.c

```diff
@@ -2254,6 +2254,33 @@ int moduleIsModuleCommand(void *module_handle, struct serverCommand *cmd) {
     return (cp->module == module_handle);
 }
 
+/* ValkeyModule_UpdateRuntimeArgs can be used to update the values of module->loadmod
+ * so that the updated values can be saved into conf file once 'config rewrite' command
+ * is called
+ * One example can be found in file modules/moduleparameter.c
```

Comment:
Good, but I have some small comments:

`module->loadmod` is something internal in Valkey. Module authors are not aware of this. Let's assume module authors are know about the API documentation and `valkeymodule.h`.

Usually we write commands in uppercase, like CONFIG REWRITE.

Dot is missing after "is called".

Is the example in src/modules/ or tests/modules/ ? We have both, so it's better to be specific. Also it can be good mention that that it is in the Valkey source code, because the documentation is published in other places and module authors are maybe not even looking at valkey source code.